app/presenters/publishing_api/historical_account_presenter.rb

```diff
@@ -60,10 +60,7 @@ def related_pms
       return [] unless all_appointees.include?(person_to_present)
 
       neighbouring_role_holders(all_appointees, person_to_present).map do |person|
-        {
-          "title" => person.name,
-          "base_path" => person.historical_accounts.present? ? person.historical_accounts.for_role(role.id).first.public_path : HistoricalAccountsIndexPresenter.base_path,
-        }
+        person.historical_accounts.present? ? person.historical_accounts.for_role(role.id).first.content_id : HistoricalAccountsIndexPresenter.content_id
```

Comment:
Just thinking about this a bit more, `HistoricalAccountsIndexPresenter.content_id` will mean we display "Past Prime Ministers" as the link text for those without a historical account, instead of their name.  This could potentially appear multiple times in the list of related links.

Maybe in these cases, we should just exclude them and show less than 5 links?  It might mean `neighbouring_role_holders` could be simplified too?